config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -202,9 +206,96 @@ config IEEE802154_NRF5_RX_STACK_SIZE
 # Enable OpenThread
 
 config NET_L2_OPENTHREAD
+    bool
+    default y if !CHIP_WIFI_NRF
+
+if NET_L2_OPENTHREAD
+
+config OPENTHREAD_PANID
+    int
+    default 4660
+
+config OPENTHREAD_CHANNEL
+    int
+    default 15
+
+config OPENTHREAD_NETWORK_NAME
+    string
+    default "OpenThread"
+
+config OPENTHREAD_XPANID
+    string
+    default "11:11:11:11:22:22:22:22"
+
+endif
+
+if CHIP_WIFI_NRF
+
+config FLASH
     bool
     default y
 
+config SETTINGS
```

Comment:
I think `SETTINGS`, `NVS` and `FLASH` could even be implied by `config CHIP` as it's not possible to build CHIP stack without persistent storage.